docs/remove-specified-tags-from-all-products/README.md

```diff
@@ -0,0 +1,45 @@
+# Remove Tag from All Products
```

Comment:
To comply with the case pattern in the library for task names, use this instead:

```md
Remove tag from all products
```